packages/medusa/src/workflows/functions/create-prducts.ts

```diff
@@ -0,0 +1,12 @@
+import { MedusaContainer, ProductTypes } from "@medusajs/types"
```

Comment:
todo: incorrect filename (we seem to have mixed up create and remove - see other comment)

Reply:
same https://github.com/medusajs/medusa/pull/4473